includes/class-wc-stripe-payment-method-configurations.php

```diff
@@ -58,29 +58,65 @@ class WC_Stripe_Payment_Method_Configurations {
 	 * @return object|null
 	 */
 	private static function get_primary_configuration( $force_refresh = false ) {
-		if ( ! $force_refresh ) {
+		// Only allow fetching payment configuration once per minute.
+		$fetch_cooldown = defined( 'IS_TESTING' ) ? 0 : get_option( 'wcstripe_payment_method_config_fetch_cooldown', 0 );
+		if ( ! $force_refresh && $fetch_cooldown > time() ) {
```

Comment:
I think that we should use `||` here. I understand that this may result in the admin page being a bit stale. But:

1. We can find ways to bust this cache with other methods.
2. Without it, do we have any rate limiting of the requests here? So, what happens if someone's browser is just sitting there making a ton of  /wp-json/wc/v3/stripe/settings.